manifests/0000_70_dns-operator_00.crd.yaml

```diff
@@ -3,9 +3,9 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.openshift.io: https://github.com/openshift/api/pull/475
+    api.openshift.io/merged-by-featuregates: "true"
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
-    include.release.openshift.io/single-node-developer: "true"
```

Comment:
So, this annotation got dropped?  We might need to notify someone about this if it was unplanned.

Reply:
Oh, it was removed on purpose: https://github.com/openshift/api/pull/1827